roles/openshift_facts/library/openshift_facts.py

```diff
@@ -56,6 +56,14 @@ def migrate_docker_facts(facts):
     if 'node' in facts and 'portal_net' in facts['node']:
         facts['docker']['hosted_registry_insecure'] = True
         facts['docker']['hosted_registry_network'] = facts['node'].pop('portal_net')
+
+    # log_options was originally meant to be a comma separated string, but
+    # we now prefer an actual list, with backward compatability:
+    if 'log_options' in facts['docker'] and \
+            isinstance(facts['docker']['log_options'], basestring) and \
+            facts['docker']['log_options'] != '':
```

Comment:
Is this test for `!= ''` still needed with the change for removing [''] as an empty value?